src/components/App/App.js

```diff
@@ -2,22 +2,37 @@ import './App.css';
 import movieData from '../../assets/movieData'
 import React, {Component} from "react"
 import Movies from '../Movies/Movies'
+import MovieDetails from '../MovieDetails/MovieDetails'
 
 class App extends Component {
   constructor() {
     super();
     this.state = {
-      movies: []
+      movies: [],
+      movie: "",
```

Comment:
Is the `movie` going to be a string once it's populated? My guess is that it will be an object, so you probably want a piece of initial state that's an empty object!